### ISSUE TITLE
ROCm 6.0 install instructions do not work

### ISSUE THREAD

**Issue by baryluk** [ROCm/ROCm#2739]
### Suggestion Description

The link to the instructions I used

https://rocm.docs.amd.com/projects/install-on-linux/en/develop/how-to/native-install/ubuntu.html#

![image](https://github.com/ROCm/ROCm/assets/110322/e64a64b0-0dd6-4559-8f7f-a961603b3c58)

There is some weird typo with this `done`.

There is no `sudo apt update` after second part, (and before doing `apt install  ...`)

Repos are broken:

```shell
Ign:5 https://repo.radeon.com/rocm/apt/6.0.0 jammy InRelease                                         
Err:6 https://repo.radeon.com/rocm/apt/6.0.0 jammy Release
  404  Not Found [IP: 13.82.220.49 443]
Reading package lists... Done
E: The repository 'https://repo.radeon.com/rocm/apt/6.0.0 jammy Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
```

this is after adding the repo and keys:

```
$ file /etc/apt/keyrings/rocm.gpg 
/etc/apt/keyrings/rocm.gpg: OpenPGP Public Key Version 4, Created Mon Aug  1 20:29:20 2016, RSA (Encrypt or Sign, 4096 bits); User ID; Signature; OpenPGP Certificate
```

**Comment by baryluk** [comment]
![image](https://github.com/ROCm/ROCm/assets/110322/ccdf3561-92d9-4009-8fcb-b4d2b5200fd5)

**Comment by YellowRoseCx** [comment]
The error doesnt appear in the official 6.0.0 documents here: https://rocm.docs.amd.com/projects/install-on-linux/en/docs-6.0.0/how-to/native-install/ubuntu.html

But the error is present in that future release develop branch link you posted

**Comment by xiaobo1025** [comment]
Hello, I have recently failed to install ROCm on my windaos10 system. Can you tell me how to install it? 
Thank you very much.

**Comment by baryluk** [comment]
> Hello, I have recently failed to install ROCm on my windaos10 system. Can you tell me how to install it? Thank you very much.

You cannot install ROCm on Windows. Please do not post unrelated issues on this issue. Open separate issue.

**Comment by xiaobo1025** [comment]
ok,thank you!

**Comment by YellowRoseCx** [comment]
> > Hello, I have recently failed to install ROCm on my windaos10 system. Can you tell me how to install it? Thank you very much.
> 
> You cannot install ROCm on Windows. Please do not post unrelated issues on this issue. Open separate issue.

The HIP SDK for ROCm was released for Windows in July, allowing Windows users to use various features like rocBLAS on Windows, so while full ROCm support isn't released yet, ROCm features are available on Windows. In an upcoming ROCm release there will be MiOpen support for Windows too, the commit to build AMD's Machine Intelligence Library on Windows was released the other day https://github.com/ROCm/MIOpen/commit/ffe4f8d24faae71d8a8d5e08d073a8c1281e2ae8



> Hello, I have recently failed to install ROCm on my windaos10 system. Can you tell me how to install it? Thank you very much.

The main installation guide is here: https://rocm.docs.amd.com/projects/install-on-windows/en/latest/
All you basically need to do is install this software: https://www.amd.com/en/developer/resources/rocm-hub/hip-sdk.html

Make sure you have a compatible GPU: https://rocm.docs.amd.com/projects/install-on-windows/en/latest/reference/system-requirements.html

**Comment by ppanchad-amd** [comment]
@baryluk Internal ticket has been created to fix documentation. Thanks!

**Comment by harkgill-amd** [comment]
Hi @baryluk, the install instructions have since been fixed for ROCm 6.0.0 and onwards. At the [Ubuntu native installation](https://rocm.docs.amd.com/projects/install-on-linux/en/docs-6.0.0/how-to/native-install/ubuntu.html) page for ROCm 6.0.0 
1. The unnecessary `done` was removed.
2. repo links were updated to the correct `6.0` paths. 
3. `sudo apt update` was added at the end of the [Register ROCm packages](https://rocm.docs.amd.com/projects/install-on-linux/en/docs-6.0.0/how-to/native-install/ubuntu.html#register-rocm-packages) code block.

Thanks for all your help!
